src/Margin.php

```diff
@@ -28,8 +28,11 @@ public function getMargin() : int
     {
         return $this->margin;
     }
-
-    public function __toString()
+    public function setMargin($margin) : void
```

Comment:
It's kind of confusing having a Margin object with a 'margin' int attribute. Is there a better name for this variable (and the associated accessors & mutators)? The margin int is supposed to express the mathematical difference between the number of votes in favor of the Margin object's winner and the number of votes in favor of the Margin object's loser.

So maybe it should be called "difference" ? I'm open to other ideas.